src/core.ts

```diff
@@ -1129,6 +1129,8 @@ export const COMMANDS: {
       if (!name) return "--> Variable name required for GET <--";
       return typeof smartBlocksContext.variables[name] === "undefined"
         ? `--> Variable ${name} not SET <--`
+        : smartBlocksContext.variables[name].includes(",")
+        ? smartBlocksContext.variables[name].split(",")
```

Comment:
@mdroidian Realized today that this breaks normal `GET`. 

Here's an example of what's expected:
![Screen Shot 2023-04-06 at 11 13 23 AM](https://user-images.githubusercontent.com/7143571/230422661-f10d8fcb-4882-4729-8368-17024e585095.png)

Here's what actually happens:
![Screen Shot 2023-04-06 at 11 12 23 AM](https://user-images.githubusercontent.com/7143571/230422506-e4790a9a-aa57-43ce-88d8-b0de648c818b.png)

Reply:
Hacked together a quick fix: https://github.com/dvargas92495/roamjs-smartblocks/pull/74